Database/schema.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS price_readings, subscriptions, users, products;
+
+CREATE TABLE products(
+    product_id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    url TEXT NOT NULL,
+    product_code TEXT NOT NULL,
+    product_name TEXT NOT NULL,
+    CONSTRAINT product_url_unique UNIQUE (url)
+);
+
+CREATE TABLE users (
+    user_id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    email TEXT NOT NULL,
+    CONSTRAINT user_email_unique UNIQUE (email)
+);
+
+CREATE TABLE subscriptions (
+    subscription_id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    user_id INTEGER NOT NULL REFERENCES users (user_id),
+    product_id INTEGER NOT NULL REFERENCES products (product_id),
+    price_threshold MONEY
```

Comment:
Is there a difference between INT and INTEGER? If not, maybe it's best to be consistent and use 1 throughout.

Reply:
Thanks, now changed all to INTEGER